index.html

```diff
@@ -6,6 +6,19 @@
 <div class="home">
   <div class="intro-header">
       <h2>We are building a mesh network in Toronto.</h2>
+    {% capture nowyear %}{{'now' | date: '%Y'}}{% endcapture %}
+    {% capture nowday %}{{'now' | date: '%j'}}{% endcapture %}
+    {% for announcement in site.announcements reversed %}
+      {% capture announceyear %}{{announcement.dateUntil | date: '%Y'}}{% endcapture %}
+      {% capture announceday %}{{announcement.dateUntil | date: '%j'}}{% endcapture %}
+      {% assign announceday = announceday | plus: 0 %}
+      {% assign nowday = nowday | minus: 2 %}
+      {% if announceyear > nowyear or announceday >= nowday and announceyear >= nowyear %}
```

Comment:
yeah, I mirrored this off of how another site handles events... 
wasn't sure if that would also be valuable for announcements (ie having recently past ones show up), I think it is 50/50 in terms of usefulness